azimuth/config.py

```diff
@@ -131,12 +170,15 @@ class DatasetWarningsOptions(BaseModel):
 class SyntaxOptions(BaseModel):
     short_sentence_max_token: int = 3
     long_sentence_min_token: int = 16
+    spacy_model: str = ""  # Language-based default value set by AzimuthConfig
```

Comment:
So, "set by AzimuthConfig" is already outdated. 😅 To avoid that, and because I don't think it's necessary, maybe we can remove that part of the comment. If a dev wants to know where/how the defaults are set, he can search for the field name. What do you think?
```suggestion
    spacy_model: str = ""  # Language-based dynamic default value
```

Reply:
Yes, perfect!